components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -454,6 +478,11 @@ public boolean isShowDisplayNameInConsentPage() {
         return showDisplayNameInConsentPage;
     }
 
+    public List<String> getAllowedScopes() {
```

Comment:
Let's add a method comment

Reply:
added in 79d44c4e3bce8a5f1c57e5dce757c1bb19337031